src/components/TextField/README.md

```diff
@@ -830,32 +830,30 @@ See Apple’s Human Interface Guidelines and API documentation about accessibili
 
 <!-- content-for: web -->
 
-### Structure
-
 Screen readers convey information about text fields automatically through native HTML.
 
 - Use the `disabled` prop to add the HTML `disabled` attribute to the text field.
 - Use the `readOnly` prop to add the HTML `readonly` attribute to the text field.
-- If you use the `type` prop, then some assistive technologies adapt the software keyboard to the current task. This helps merchants with mobility, vision, and cognitive issues to enter information more easily.
+- If you use the `type` prop, then some browsers and assistive technologies adapt the software keyboard to the current task. This helps users with mobility, vision, and cognitive issues to enter information more easily.
 
 Use the `id` prop to provide a unique `id` attribute value for the text field. If you don't provide an `id`, then the component generates one automatically. All text fields need to have unique `id` values.
 
 ### Labeling
 
-The `label` prop is required to convey the purpose of the checkbox to all merchants.
+The `label` prop is required to convey the purpose of the checkbox to all users.
 
-If there are separate visual cues that convey the purpose of the text field to sighted merchants, then the label can be visually hidden with the `labelHidden` prop.
+If there are separate visual cues that convey the purpose of the text field to sighted users, then the label can be visually hidden with the `labelHidden` prop.
 
 When you provide help text via the `helpText` prop or an inline error message via the `error` prop, the help or error content is conveyed to screen reader users with the `aria-describedby` attribute. This attribute causes the content to be read along with the label, either immediately or after a short delay.
 
-Use the `placeholder` prop to provide additional instructions. However, don’t rely on placeholders alone since the content isn’t always conveyed to all merchants.
+Use the `placeholder` prop only to provide formatting instructions that aren't essential. [HTML placeholders can be difficult for users to access.](https://www.nngroup.com/articles/form-design-placeholders/) Don’t rely on placeholders to convey important information since the content isn’t always conveyed to all users.
 
 <!-- usageblock -->
 
 #### Do
 
-- Use the label to provide instructions critical to using the text field
-- Use help text and placeholder text to provide additional, non-critical instructions
+- Use the label as the primary way to provide instructions critical to using the text field
+- Use help text as a secondary way to provide additional, non-critical instructions
```

Comment:
- Should we provide examples?
- Since _label_ and _help text_ both have a "Do", would it be useful to also provide a "Do" for _placeholder_?

Reply:
My personal opinion is that placeholders are...not good. I don't want to provide examples where they are deemed useful, because I think folks should use labels and help text instead. I am open to arguments on this topic though. 😂

Reply:
I agree with that approach, Devon. And that aligns with the upcoming content pattern that I mentioned in my comment above about not using placeholder text unless it's for a search or filter bar ("Search {x}", or "Filter {x}") because it's such a common pattern. I'm assuming that the text field component isn't used in this way, so using that as a DO example here wouldn't work.

Reply:
Case where a placeholder is useful: disambiguation. Not crucial, but a nice little nudge to write the correct format in the field. For example: "https://" for a URL, or "@username" for a Twitter account. 
I don't have research data on hand for this, but I can say that it definitely helps me reduce the number of errors I do when filling a form.